crates/alerter/src/main.rs

```diff
@@ -196,10 +230,16 @@ impl Alerter {
                 (order, existing_time)
             })
             .collect::<Vec<_>>();
+
         tracing::debug!("found {} open orders", orders.len());
+
         std::mem::swap(&mut self.open_orders, &mut orders);
         // Keep only orders that were open last update and are not open this update.
-        orders.retain(|order| !self.open_orders.contains(order));
+        orders.retain(|(order, _)| {
+            self.open_orders
+                .iter()
+                .all(|(open_order, _)| open_order.uid != order.uid)
```

Comment:
Seems unrelated, but it isn't. We use order UIDs to identify the two, otherwise changes in surplus fee can cause an order to be considered a "closed" order, even if that is not the case.

Reply:
Seems unintuitive enough to warrant an inline comment.